notebooks/1_spikesorting.ipynb

```diff
@@ -4,26 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    filter_param_dict['frequency_min'] = 600
</pre><p>why are we adjusting this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/nwb_datajoint/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>I think for hippocampal the preference is 600Hz instead of 300Hz, which is the default, no? Or was your point just to explain this?</p>

Reply:
<p>OK yes maybe just include that explanation.</p>